azure-pipelines.yml

```diff
@@ -0,0 +1,74 @@
+variables:
+  MPLBACKEND: agg # Necessary when running test which generate matplotlib figure
+  ENV_NAME: test_env
+
+trigger:
+  tags:
+    include:
+    # build on any tag
+    - '*'
+  branches:
+    include:
+    # build on all branches
+    - '*'
+
+resources:
+  repositories:
+  - repository: templates
+    type: github
+    name: hyperspy/ci-scripts
+    # For more details on service connection endpoint, see
+    # https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints
+    endpoint: lumispy # Azure DevOps service connection
+
+strategy:
+  matrix:
+    Linux_Python38:
+      vmImage: 'ubuntu-latest'
+      PYTHON_VERSION: '3.8'
+#    Linux_Python37:
+#      vmImage: 'ubuntu-latest'
+#      PYTHON_VERSION: '3.7'
+#    Linux_Python36:
+#      vmImage: 'ubuntu-latest'
+#      PYTHON_VERSION: '3.6'
+    MacOS_Python38:
+      vmImage: 'macOS-latest'
+      PYTHON_VERSION: '3.8'
+#    MacOS_Python36:
+#      vmImage: 'macOS-latest'
+#      PYTHON_VERSION: '3.6'
+    Windows_Python38:
+      vmImage: 'windows-latest'
+      PYTHON_VERSION: '3.8'
+#    Windows_Python36:
+#      vmImage: 'windows-latest'
+#      PYTHON_VERSION: '3.6'
+
+pool:
+  vmImage: '$(vmImage)'
+
+steps:
+- template: azure_pipelines/clone_ci-scripts_repo.yml@templates
+- template: azure_pipelines/activate_conda.yml@templates
+- template: azure_pipelines/setup_anaconda_packages.yml@templates
+
+- bash: |
+    source activate $ENV_NAME
+    # Install package using pip
+    pip install .[tests]
```

Comment:
I would create the file `conda_environment.yml` and conda_environment_dev.yml` and replace this line with
```
pip install . --no-deps
```
The `--no-deps` will make sure you don't mix pypi and conda packages, which is not advised because there are not ABI compatible and make sure all you dependencies are on conda-forge - even if at the moment, the last point is not really relevant.

Reply:
Thank you!
Now this is resolved.